tools/protoc-gen-chpl/Makefile

```diff
@@ -32,7 +32,17 @@ linkFile=$(bdir)/protoc-gen-chpl
 SOURCES=$(wildcard *.cpp)
 HEADERS=$(wildcard *.h)
 LDFLAGS = -lprotobuf -lprotoc
-CPPFLAGS = -I.
+CPPFLAGS = -I. -std=c++11
+
+ifeq ($(CHPL_MAKE_PLATFORM),darwin)
+	PROTOC_LIB_LOC=$(dir $(shell which protoc))../lib
+	UPDATE_RPATH = install_name_tool -add_rpath $(PROTOC_LIB_LOC)
+else
+	# Don't want to accidentally execute protoc-gen-chpl when install_name_tool
+	# isn't available, so instead use a noop
+	UPDATE_RPATH = touch
```

Comment:
There are linux commands as well to update an rpath but apparently they are not needed here?
Is this an issue with system-wide installation of protoc vs a custom one?

Reply:
I encountered this with the spack build of protoc.  I think it's something about how protoc is built on systems that default to dynamic linking, but I could be wrong about that